### ISSUE TITLE
Missing file/bone errors

### ISSUE THREAD

**Issue by Blutze** [toadie2k/NIArms#138]
Selecting either one of the Mk48s, even with the M60 pack loaded, gives me a popup warning:
`Cannot load texture hlc_wp_m60e4\tex\m60e4_ti_ca.paa.`

Opening the inventory with the M14 SOPMOD gives me this one:
`Picture \hlc_wp_m14\tex\ui\gear_sopmod_mstu_ca not found`
Snooping around on the repo, I can find a `gear_sopmod_mst_ca.paa`, without the 'u'.

Selecting any rifle with that "common" M203-2003 grenade launcher while NOT having the SG550 pack loaded gives me these (yes, twice):
```
Cannot load texture nia_wp_sg550\tex\toadie_sg550\nico_nohq.paa.
Cannot load texture nia_wp_sg550\tex\toadie_sg550\nico_smdi.paa.
Cannot load texture nia_wp_sg550\tex\toadie_sg550\nico_nohq.paa.
Cannot load texture nia_wp_sg550\tex\toadie_sg550\nico_smdi.paa.
```
At least on the repo, they are cross referencing each other... the SG550s are using the files from core, and the launcher in core is using the 550 files.

Holding down-arrow on guns, loaded magazines and equipped optics (1913 only) gives me rpt entries with "No geometry and no visual shape" for these files:<details><summary>Many p3d files</summary>
a3\data_f\particleeffects\universal\ammobelt_links.p3d

hlc_core\mesh\magazines\proxies\30rnd_762x39_ak.p3d
hlc_wp_mp5\mesh\magazine\proxy\30rnd_10mm_mp510.p3d
hlc_wp_mp5\mesh\magazine\proxy\30rnd_9x19_mp5.p3d
hlc_core\mesh\magazines\proxies\30rnd_545x39_ak74_6l20.p3d
hlc_wp_mg3\mesh\magazine\proxy\50rnd_762nato_dm1.p3d
nia_wp_sg550\mesh\magazine\proxy\30rnd_556nato_sg550.p3d
hlc_wp_aug\mesh\magazine\proxy\25rnd_9x19mm_aug.p3d
hlc_core\mesh\magazines\proxies\30rnd_762x39_ak_6l10.p3d
hlc_core\mesh\magazines\proxies\30rnd_762x39_ak_pmag.p3d
hlc_core\mesh\magazines\proxies\30rnd_762x39_ak_valmet.p3d
hlc_core\mesh\magazines\proxies\40rnd_762x39_ak_6l2.p3d
hlc_core\mesh\magazines\proxies\75rnd_762x39_rpk.p3d
hlc_wp_mg3\mesh\magazine\proxy\100rnd_762nato_dm1.p3d
hlc_wp_mg3\mesh\magazine\proxy\250rnd_762nato_dm1.p3d
hlc_core\mesh\magazines\proxies\30rnd_545x39_ak74_6l23.p3d
hlc_core\mesh\magazines\proxies\30rnd_545x39_ak74_6l20plum.p3d
hlc_core\mesh\magazines\proxies\45rnd_545x39_ak74_6l18.p3d
hlc_core\mesh\magazines\proxies\45rnd_545x39_ak74_6l26.p3d
hlc_core\mesh\magazines\proxies\60rnd_545x39_ak74_6l31.p3d

nia_wp_c96\mesh\tracer_blaster.p3d

hlc_core\mesh\accessories\sights\reticles\reticle_10xmildot.p3d
hlc_core\mesh\accessories\sights\reticles\fc-dm_atacr_1x.p3d
hlc_core\mesh\accessories\sights\reticles\fc-dm_atacr_2x.p3d
hlc_core\mesh\accessories\sights\reticles\fc-dm_atacr_3x.p3d
hlc_core\mesh\accessories\sights\reticles\fc-dm_atacr_4x.p3d
hlc_core\mesh\accessories\sights\reticles\fc-dm_atacr_5x.p3d
hlc_core\mesh\accessories\sights\reticles\fc-dm_atacr_6x.p3d
hlc_core\mesh\accessories\sights\reticles\fc-dm_atacr_7x.p3d
hlc_core\mesh\accessories\sights\reticles\fc-dm_atacr_8x.p3d
</details>
As well as these boney errors:<details><summary>Some bones</summary>
MG3skel: Invalid parent bone 'belt011' for 'bullet11'
nia_MG3mag_skel: Invalid parent bone 'feedtray' for 'magazine'
nia_MG3mag_skel: Invalid parent bone 'belt011' for 'bullet11'
MG42skel: Invalid parent bone 'belt011' for 'bullet11'
nia_MG42mag_skel: Invalid parent bone 'belt011' for 'bullet11'
: Invalid parent bone 'belt011' for 'bullet11'
: Invalid parent bone 'feedtray' for 'magazine'
</details>

Other than the M14 (and potentially the MG3/MG42, haven't fired them yet), none of these seem to be actually causing visible issues. Even the Mk48s that complain about their TI texture missing are heating up nicely under thermals.

**Comment by toadie2k** [comment]
Yeah, I think that the p3d errors are just arma trying to project to LODs that have no functional purpose in their use case (see: Reticle mesh needing a Hitbox/Geometry LOD....what?) 

The Boney error's legit though. I'll push the change to github.

**Comment by Blutze** [comment]
The sights' geometry might be relevant by association (the power of OOP compels you!) with `useModelOptics` + iron sights + LODs. And the mags should have a hitbox, they block bullets. At least the RHS PKM did... but the MXs don't anymore? IDK.

Nico nico kneekaps files are still complaining. Rest in this ticket is behind the scenes.
